frontend/src/components/ResourceActionDialog.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <v-dialog v-model="isOpen" id="resource-action-dialog" width="500">
```

Comment:
```suggestion
  <v-dialog v-model="isOpen" width="500">
```
est-ce que `width="500"` marche pour la version mobile aussi ?

Reply:
j'ai renommé à max-width pour être plus clair + testé sur mobile, c'est responsive :+1: